at-luantran/Week5/SinatraApp/migrate.rb

```diff
@@ -0,0 +1,44 @@
+require 'mysql2'
+
+connect = Mysql2::Client.new(host: 'localhost', username: 'root',
+                             password: 'hoichuphong513')
+
+create_db = 'CREATE DATABASE IF NOT EXISTS LuanTran_Sinatra'
+create_users = 'CREATE TABLE IF NOT EXISTS `LuanTran_Sinatra`.`users` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NULL,
+  `address` TEXT NULL,
+  PRIMARY KEY (`id`));'
+create_posts = 'CREATE TABLE IF NOT EXISTS `LuanTran_Sinatra`.`posts` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `content` TEXT NULL,
+  `user_id` TEXT NULL,
+  `comment_count` INT NULL,
+  PRIMARY KEY (`id`));'
+create_comments = 'CREATE TABLE IF NOT EXISTS `LuanTran_Sinatra`.`comments` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `content` TEXT NULL,
+  `post_id` TEXT NULL,
+  PRIMARY KEY (`id`));'
+
+connect.query(create_db)
+connect.query(create_users)
+connect.query(create_posts)
+connect.query(create_comments)
+
+create_user = "INSERT INTO `LuanTran_Sinatra`.`users` (`name`,
+              `address`) VALUES ('admin', 'Da Nang');"
+connect.query(create_user)
+1.upto(20) do |x|
+  create_post = "INSERT INTO `LuanTran_Sinatra`.`posts` (`content`,
+                `user_id`, `comment_count`) VALUES ('Content #{x}', 1, 0);"
+  connect.query(create_post)
+end
+
+1.upto(20) do |x|
+  create_post = "INSERT INTO `LuanTran_Sinatra`.`comments` (`content`,
+                `post_id`) VALUES ('Comment #{x}', 1);"
+  count_comment = "UPDATE `LuanTran_Sinatra`.`posts` SET comment_count = comment_count + 1 WHERE id = 1"  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/0b1d0ee0840c9861a2ecd6316ae3e7126fae5eab/at-luantran/Week5/SinatraApp/migrate.rb#L41"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>